Levi9.ERP/Controllers/ClientController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using AutoMapper;
+using Levi9.ERP.Domain.Models.DTO;
+using Levi9.ERP.Domain.Services;
+using Levi9.ERP.Requests;
+using Levi9.ERP.Responses;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Levi9.ERP.Controllers
+{
+    [Route("api/client")]
+    [ApiController]
+    [Produces("application/json", "application/xml")]
+    public class ClientController : ControllerBase
+    {
+        private readonly IClientService _clientService;
+        private readonly IMapper _mapper;
+        private readonly IUrlHelper _urlHelper;
+
+        public ClientController(IClientService _clientService, IMapper _mapper, IUrlHelper _urlHelper)
+        {
+            this._clientService = _clientService;
```

Comment:
Constructor needs to be changed:
 public ClientController(IClientService clientService, IMapper mapper, IUrlHelper urlHelper)
        {
            _clientService = clientService;
            _mapper = mapper;
            _urlHelper = urlHelper;
        }